doc/lsst.ap.verify/command-line-reference.rst

```diff
@@ -129,7 +129,7 @@ Required arguments are :option:`--dataset` and :option:`--output`.
 
    .. note::
 
-      At present, ap_verify assumes that the provided pipeline includes the ``diaPipe`` task from the AP pipeline, and configures it on the fly.
+      At present, ap_verify assumes that the provided pipeline includes the ``associateApdb`` task from the AP pipeline, and configures it on the fly.
       It will likely crash if this task is missing.
```

Comment:
This paragraph is out of date. [The current code](https://github.com/lsst/ap_verify/blob/main/python/lsst/ap/verify/pipeline_driver.py#L241-L242) assumes an `apdb_config` pipeline parameter, which is a much more robust requirement (but should still be noted).